bundles/org.openhab.binding.dwdpollenflug/src/main/java/org/openhab/binding/dwdpollenflug/internal/handler/DWDPollenflugBridgeHandler.java

```diff
@@ -0,0 +1,217 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dwdpollenflug.internal.handler;
+
+import java.net.URI;
+import java.util.Set;
+import java.net.SocketTimeoutException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.HttpResponse;
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.api.Result;
+import org.eclipse.jetty.client.util.BufferingResponseListener;
+import org.eclipse.jetty.http.HttpMethod;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.dwdpollenflug.internal.DWDPollingException;
+import org.openhab.binding.dwdpollenflug.internal.config.DWDPollenflugBridgeConfiguration;
+import org.openhab.binding.dwdpollenflug.internal.dto.DWDPollenflug;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DWDPollenflugBridgeHandler} is the handler for bridge thing
+ *
+ * @author Johannes Ott - Initial contribution
+ */
+@NonNullByDefault
+public class DWDPollenflugBridgeHandler extends BaseBridgeHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(DWDPollenflugBridgeHandler.class);
+
+    private static final String DWD_URL = "https://opendata.dwd.de/climate_environment/health/alerts/s31fg.json";
+
+    private DWDPollenflugBridgeConfiguration bridgeConfig = new DWDPollenflugBridgeConfiguration();
+
+    private @Nullable ScheduledFuture<?> pollingJob;
+
+    private @Nullable DWDPollenflug pollenflug;
+
+    private final Set<DWDPollenflugRegionHandler> regionListeners = ConcurrentHashMap.newKeySet();
+
+    private final HttpClient client;
+
+    private final Gson gson = new Gson();

```

Comment:
can you remove all the blank lines between these fields. 
Also can you make sure to order the fields: static final ->final->the rest.

Reply:
Removed and sorted